UnitsNet.Serialization.JsonNet/UnitsNet.Serialization.JsonNet.csproj

```diff
@@ -42,8 +42,8 @@
   <!-- NuGet references that work for both signed and unsigned -->
   <ItemGroup>
     <PackageReference Include="JetBrains.Annotations" Version="2019.1.3" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
```

Comment:
Nuget version upgrades could benefit from its own pull request so it is more visible in the git history and searchable in the github issue list. Would you mind moving it to a separate PR?

Reply:
No worries, just old habits to keep packages updated. Will revert back to original versions and add new PR for stable NuGet package updates,